js/ui/date_box/ui.date_box.mask.js

```diff
@@ -175,6 +175,7 @@ var DateBoxMask = DateBoxBase.inherit({
     _optionChanged: function(args) {
         switch(args.name) {
             case "useMaskBehavior":
+                this._clearState();
```

Comment:
Remove clearState. This method already calls in the [renderMask](https://github.com/DevExpress/DevExtreme/pull/4698/files#diff-e23fb8153c04867cc50fe7f201618494R49) method

Reply:
Ok, I will remove this.